packages/selfcare-onboarding-consumer/test/selfcareOnboardingProcessor.test.ts

```diff
@@ -264,10 +264,10 @@ describe("Message processor", () => {
       })
     );
   });
-  it.each(["SCP", "PRV"])(
+  it.each(["SCP", "PRV", "PT"])(
     "should upsert tenant with institutionType %s correctly",
     async (institutionType) => {
-      const origin = "PDND_INFOCAMERE";
+      const origin = "*INFOCAMERE";
```

Comment:
I admit the `*` confused me for a moment.
Do you consider it necessary?
As I understood: the star has no specific meaning because origins should match the `allowedOrigins` configuration (which is application independent), but I thought it would involve a regex

Reply:
Yes it basically checks that the suffix is added to the given origin, the `*` shouldn't be needed